shell/browser/electron_navigation_throttle.cc

```diff
@@ -37,11 +39,26 @@ ElectronNavigationThrottle::WillStartRequest() {
     return PROCEED;
   }
 
-  if (handle->IsRendererInitiated() &&
+  bool is_renderer_initiated = handle->IsRendererInitiated();
+
+  // Chrome's internal pages always mark navigation as browser-initiated. Let's
+  // ignore that.
+  // https://source.chromium.org/chromium/chromium/src/+/main:content/browser/renderer_host/navigator.cc;l=883-892;drc=0c8ffbe78dc0ef2047849e45cefb3f621043e956
+  if (!is_renderer_initiated &&
+      ui::PageTransitionIsWebTriggerable(handle->GetPageTransition())) {
```

Comment:
(This was new to me, so tldr for other reviewers: [true if it's a link, auto subframe, manual subframe, or form submission"](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/page_transition_types.cc;l=64;drc=0c8ffbe78dc0ef2047849e45cefb3f621043e956?q=PageTransitionIsWebTriggerable))